installer/pkg/components/components.go

```diff
@@ -83,6 +82,7 @@ var FullHelmDependencies = common.CompositeHelmFunc(
 var CommonObjects = common.CompositeRenderFunc(
 	dockerregistry.Objects,
 	cluster.Objects,
+	gitpod.Objects,
```

Comment:
This is moved into the common objects which, truthfully, is where it should have been all along